### DISCUSSION TITLE
Open port on for MYSQL database connection

### DISCUSSION THREAD

**Discussion by kvyeap** [NginxProxyManager/nginx-proxy-manager#4212] [Q&A]
Hi guys, I am beginner to web hosting on VPS and recently setting up my services with NPM. I bumped into an issue with setting up dockerized MYSQL database and I am wondering if I could make it running as a subdomain like `mysql.domain.com` or perhaps with an exposed port like `mysql.domain.com:3306` so it's somehow looking better and easier for my developers to remember the connection. 

I don't know if the idea is correct or not. Would appreciate if anyone could share some advice to me. Thanks

**Comment by lastsamurai26** [comment]
It is never good to put a database on the Internet without a backup, so I would not approve of this, but you can hide the port of a Docker Mysql server by not publishing it but using another one.

For example, you remove ports: completely from the Docker compose and open port 10932 via the NPM and forward it internally to the mysql server, otherwise this would not work in my opinion.
